code/modules/mob/living/carbon/human/species/skeleton.dm

```diff
@@ -10,6 +10,9 @@
 	default_language = "Galactic Common"
 	unarmed_type = /datum/unarmed_attack/punch
 
+	greater_form = null
+	primitive_form = null // A skeleton is the perfect form, what need is there for others?
```

Comment:
Oh! Default is null! I assumed it was set in the base species.dm - I'll fix this now.